packages/orbit-components/src/Separator/Separator.stories.tsx

```diff
@@ -8,6 +8,16 @@ import Text from "../Text";
 
 import Separator from ".";
 
+const FlightDirect = () => (
+  <svg
+    className="orbit-icon size-icon-medium inline-block shrink-0 self-center fill-current align-middle"
+    viewBox="0 0 24 24"
+    preserveAspectRatio="xMidYMid meet"
+  >
+    <path d="M17.043 8.683a.588.588 0 0 1 .852 0l2.922 2.921c.242.244.242.609 0 .792l-2.922 2.922a.554.554 0 0 1-.426.182c-.366 0-.609-.243-.609-.609V12.9L3.9 12.9a.9.9 0 0 1-.893-.787L3 12a.9.9 0 0 1 .9-.9l12.96-.001v-1.99c0-.146.039-.254.117-.353l.066-.073Z" />
+  </svg>
+);
+
```

Comment:
What's preventing us from importing the icon directly?

Is it because it can lead to side-effects if the icon changes?

Reply:
Chose the "import icon" approach in the end.

Reply:
I agree, we will detect it anyway, and icons don't change that often.

Reply:
Resolving the comment then, thanks Marco!